coffeecard/CoffeeCard.WebApi/Services/EmailService.cs

```diff
@@ -99,15 +91,10 @@ public void SendVerificationEmailForChangedEmail(User user, string token, string
         public void SendVerificationEmailForLostPw(User user, string token)
```

Comment:
Rename to remove the verification part

Reply:
While easily done, I'm not sure I understand why you want to change it? It still serves to verify that the user requested a new password

Reply:
It's just hairsplitting, I'll approve the PR without this. My point was that I do not see the email as a verification email, more like a choose a new password email, hence we could remove the verification word :)